src/main/scala/scalismo/statisticalmodel/StatisticalVolumeIntensityModel.scala

```diff
@@ -0,0 +1,100 @@
+package scalismo.statisticalmodel
+
+import breeze.linalg.DenseVector
+import scalismo.common._
+import scalismo.geometry._
+import scalismo.mesh._
+import scalismo.utils.Random
+
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe.TypeTag
+
+trait StatisticalVolumeIntensityModel[S] {
```

Comment:
What is the reason that it is split into a trait and an implementation class? Do we expect other subclasses? Would it even be possible to have other subclasses, which satisfy these type signatures?